### PR TITLE
updates to TF BSL Update _index.md

### PR DESCRIPTION
**Pull request by interurban**
Clarify TF's adoption of BSL

## Checklist:

- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/docs/concepts/vs/terraform/_index.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Terraform provides open-source [infrastructure as code](/what-is/what-is-infrast
...
+Terraform and Pulumi have some similarities in that you can create, deploy and manage infrastructure as code on any cloud. Terraform has adopted a Business Source License model, while Pulumi is [open source on GitHub](https://github.com/pulumi/pulumi) and is [free to use](/docs/get-started/). Both Terraform and Pulumi offer a desired state infrastructure as code model where the code represents the desired infrastructure state and the deployment engine compares this desired state with the stack’s current state and determines what resources need to be created, updated or deleted.  Both Terraform and Pulumi support many cloud providers, including [AWS](/aws/), [Azure](/azure/), and [Google Cloud](/gcp/), plus other services like CloudFlare, Digital Ocean, and more. Thanks to integration with Terraform providers, Pulumi is also able to support a superset of the providers that Terraform currently offers.
```
Comment (usrbinkat) [inline themes/default/content/docs/concepts/vs/terraform/_index.md:83]: I'm going to suggest that licensing discussion should be #1 on the list of "key differences". This is now a key differentiator that we will be turning up the volume on.

File: themes/default/content/docs/concepts/vs/terraform/_index.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Pulumi’s universal infrastructure as code platform helps teams tame the cloud
 
 ## What is Terraform?
 
-Terraform provides open-source [infrastructure as code](/what-is/what-is-infrastructure-as-code/) software for cloud service management with a consistent CLI workflow. Terraform allows you to write, plan, and apply changes to deliver infrastructure as code.
+Terraform provides [infrastructure as code](/what-is/what-is-infrastructure-as-code/) software for cloud service management with a consistent CLI workflow. Terraform allows you to write, plan, and apply changes to deliver infrastructure as code.
```
Comment (usrbinkat) [inline themes/default/content/docs/concepts/vs/terraform/_index.md:36]: Perhaps something like:

"Terraform is a free, source available infrastructure as code tool from HashiCorp" ...  bla bla bla ...

**Review by usrbinkat — COMMENTED** [review]
wording re-org and refinement

File: themes/default/content/docs/concepts/vs/terraform/_index.md (hunk 1 above)
Comment (interurban) [reply themes/default/content/docs/concepts/vs/terraform/_index.md:83]: Great call out!

**Comment by interurban** [comment]
@usrbinkat I incorporated your feedback, and made the licensing model #3 on the list :) let me know if your okay with the rest of this and approve and I'll merge.

File: themes/default/content/docs/concepts/vs/terraform/_index.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ Terraform provides open-source [infrastructure as code](/what-is/what-is-infrast
...
+Terraform and Pulumi have some similarities in that you can create, deploy and manage infrastructure as code on any cloud. Terraform is a free, source available infrastructure as code tool from HashiCorp, while Pulumi is [open source on GitHub](https://github.com/pulumi/pulumi) and is [free to use](/docs/get-started/). Both Terraform and Pulumi offer a desired state infrastructure as code model where the code represents the desired infrastructure state and the deployment engine compares this desired state with the stack’s current state and determines what resources need to be created, updated or deleted.  Both Terraform and Pulumi support many cloud providers, including [AWS](/aws/), [Azure](/azure/), and [Google Cloud](/gcp/), plus other services like CloudFlare, Digital Ocean, and more. Thanks to integration with Terraform providers, Pulumi is also able to support a superset of the providers that Terraform currently offers.
```
Comment (usrbinkat) [inline themes/default/content/docs/concepts/vs/terraform/_index.md:83]:
```suggestion
Similarities between Terraform and Pulumi include the ability to create, deploy, and manage infrastructure as code on any cloud. Terraform is a free-to-use, source available, infrastructure as code tool from HashiCorp. Pulumi is [and will remain](https://www.pulumi.com/blog/pulumi-hearts-opensource/) fully open source [Apache 2.0 licensed](https://github.com/pulumi/pulumi). Both Terraform and Pulumi follow a desired state infrastructure as code model, where the IaC code represents the desired state of the infrastructure. The deployment engine compares this desired state with the current state of the stack and determines the necessary actions, such as creating, updating, or deleting resources. Both Terraform and Pulumi support many cloud providers, including [AWS](/aws/), [Azure](/azure/), and [Google Cloud](/gcp/), plus other services like CloudFlare, Digital Ocean, and more. Finally, with Pulumi you get access to all Pulumi providers as well as support for all open source terraform providers.
```

File: themes/default/content/docs/concepts/vs/terraform/_index.md [hunk 4]
```diff
@@ -80,7 +80,7 @@ Terraform provides open-source [infrastructure as code](/what-is/what-is-infrast
...
+Similarities between Terraform and Pulumi include the ability to create, deploy, and manage infrastructure as code on any cloud. Terraform is a free-to-use, source available, infrastructure as code tool from HashiCorp. Pulumi is [and will remain](https://www.pulumi.com/blog/pulumi-hearts-opensource/) fully open source [Apache 2.0 licensed](https://github.com/pulumi/pulumi). Both Terraform and Pulumi follow a desired state infrastructure as code model, where the IaC code represents the desired state of the infrastructure. The deployment engine compares this desired state with the current state of the stack and determines the necessary actions, such as creating, updating, or deleting resources. Both Terraform and Pulumi support many cloud providers, including [AWS](/aws/), [Azure](/azure/), and [Google Cloud](/gcp/), plus other services like CloudFlare, Digital Ocean, and more. Finally, with Pulumi you get access to all Pulumi providers as well as support for all open source terraform providers.
```
Comment (aaronkao) [inline themes/default/content/docs/concepts/vs/terraform/_index.md:83]: When we say "free to use, source available" why aren't we calling out specifically that it is Business Source License?

File: themes/default/content/docs/concepts/vs/terraform/_index.md [hunk 5]
```diff
@@ -92,11 +92,11 @@ Here is a summary of the key differences between Pulumi and Terraform:
 | ------- | ------ | --------- |
 | [Language Support](#language) | Python, TypeScript, JavaScript, Go, C#, F#, Java, YAML | HashiCorp Configuration Language (HCL) |
 | [IDE Support](#ide) | Code completion, strong typing, error squiggles, rich resource documentation, etc. | Limited |
+| [OSS License](#license) | Apache License 2.0 | Business Source License 1.1 |
```
Comment (joeduffy) [inline themes/default/content/docs/concepts/vs/terraform/_index.md:95]: Nit, BUSL is not an "OSS License", so the row label doesn't work perfectly. Maybe:

```
[Open Source] | Yes, Apache 2.0 | No, Business Source License 1.1 |
```

File: themes/default/content/docs/concepts/vs/terraform/_index.md (hunk 4 above)
Comment (joeduffy) [inline themes/default/content/docs/concepts/vs/terraform/_index.md:83]: "and will remain" feels preemptively and awkwardly defensive to me.

Comment (interurban) [reply themes/default/content/docs/concepts/vs/terraform/_index.md:83]: Thx Joe I'll incorporate this feedback!

**Review by cnunciato — APPROVED** [review]
Dropping a ✅ so y'all have it when you need it.

**Pull request merged**